packages/cryptsetup.rb

```diff
@@ -0,0 +1,29 @@
+require 'package'
+
+class Cryptsetup < Package
+  description 'Cryptsetup is utility used to conveniently setup disk encryption based on DMCrypt kernel module.'
```

Comment:
Some might say `Cryptsetup is an utility...` would be the proper grammar but I actually prefer `Cryptsetup is a utility...`.  You could also say `The cryptsetup utility is...`.  Either way would be fine.